lib/specinfra/ext/class.rb

```diff
@@ -2,7 +2,7 @@ class Class
   def subclasses
     result = []
     ObjectSpace.each_object(Class) do |k|
-      next if k.name.nil?
+      next unless k == k.ancestors.first #Skips if this is a singleton class
```

Comment:
Sounds great @mizzy ! Sorry if I'm rushing, but when can we expect a 2.59.3 release?